cluster/manifests/deployment-service/01-config.yaml

```diff
@@ -20,6 +21,10 @@ data:
   oidc-provider-arn: "arn:aws:iam::{{accountID .Cluster.InfrastructureAccount}}:oidc-provider/{{.Cluster.LocalID}}.{{.Values.hosted_zone}}"
   oidc-subject-key: "{{.Cluster.LocalID}}.{{.Values.hosted_zone}}:sub"
 {{- end }}
+=======
+  oidc-provider-arn: "arn:aws:iam::{{accountID .Cluster.InfrastructureAccount}}:oidc-provider/{{.Cluster.LocalID}}.{{.Values.hosted_zone}}"
+  oidc-subject-key: "{{.Cluster.LocalID}}.{{.Values.hosted_zone}}:sub"
+>>>>>>> dev
```

Comment:
This part I already pushed in a merge to the `eks` branch. Can you check if you have the latest `eks` branch locally?

Reply:
Was  fixed by the bot